### ISSUE TITLE
Feature request:TL implement of CornerPooling

### ISSUE THREAD

**Issue by Windaway** [tensorlayer/tensorlayer#782]
`class CornetPool(Layer):
    """The :class:`CornetPool` class is 2D cornet pool, see `here <https://arxiv.org/abs/1808.01244/>`__.

    Parameters
    --------------
    prev_layer : :class:`Layer`
        Previous layer.
    filter_size : int
        The filter size.
    mode:str
        BottomRight for the top left corrnet,
        TopLeft for the bottom right corrnet.
    name : str
        A unique layer name.
    """

    def __init__(
        self,
        prev_layer = None,
        filter_size=(3,3),
        mode='BottomRight',
        name ='cornerpool_layer',
    ):
        Layer.__init__(self, prev_layer=prev_layer, name=name)
        self.inputs = prev_layer.outputs
        if mode=='BottomRight':
            temp=tf.keras.layers.ZeroPadding2D(padding=((0, filter_size[0]-1), (0,  filter_size[1]-1)), name=name)(self.inputs)
            temp=tf.layers.max_pooling2d(temp, (filter_size[0],1), (1,1), padding='valid', data_format='channels_last')
            self.outputs=tf.layers.max_pooling2d(temp, (1,filter_size[1]), (1,1), padding='valid', data_format='channels_last',name=name)
        elif mode=='TopLeft':
            temp=tf.keras.layers.ZeroPadding2D(padding=((filter_size[0]-1,0), (filter_size[1]-1,0)), name=name)(self.inputs)
            temp=tf.layers.max_pooling2d(temp, (filter_size[0],1), (1,1), padding='valid', data_format='channels_last')
            self.outputs=tf.layers.max_pooling2d(temp, (1,filter_size[1]), (1,1), padding='valid', data_format='channels_last',name=name)
        else:
            raise AssertionError("Mode should be of 'BottomRight'and'TopLeft' ")
        self._add_layers(self.outputs)`

Maybe Zero Padding is not a rigorous method for leaky relu activation func?

**Comment by DEKHTIARJonathan** [comment]
Duplicate of https://github.com/tensorlayer/tensorlayer/issues/781
